test/spec/checkAndEvaluateMath.spec.ts

```diff
@@ -31,19 +42,24 @@ describe('check and evaluate math', () => {
 
   it('can evaluate math expressions where more than one token has a unit, assuming no mixed units are used', () => {
     expect(checkAndEvaluateMath({ value: '4em + 7em', type: 'dimension' })).to.equal('11em');
-    expect(checkAndEvaluateMath({ value: '4 + 7rem', type: 'dimension' })).to.equal('4 + 7rem');
-    expect(checkAndEvaluateMath({ value: '4em + 7rem', type: 'dimension' })).to.equal('4em + 7rem');
+    expect(() => checkAndEvaluateMath({ value: '4 + 7rem', type: 'dimension' })).to.throw(
```

Comment:
This seems fine by me but we do need to ensure we have an integration test somewhere that verifies that the token value remains unchanged if the math throws an error. So the behavior of the token value is actually the same (which is important for backwards compatibility), except this function throws now.